rpmbuild/bin/copr-builder-rhsm-subscribe

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/python3
+
+"""
+Run `subscription-manager register` without having the --activation key in
+/proc/self/environ or /proc/self/comm.
+"""
+
+import argparse
+import getpass
+import sys
+
+from subscription_manager.scripts.subscription_manager import main as rhsm
```

Comment:
I am a bit afraid of them changing the location or the internals of the script and I would probably rather call it through `subprocess` but we can do this.

Reply:
This is sub-optimal, but subprocess.call means the key goes to /proc/self/cmdline

Reply:
Can you please add a code comment about that so that I don't get the great idea to re-implement it someday? :-)

Reply:
done